src/templates.json

```diff
@@ -1,3 +1,3 @@
 {
-  "navy": "https://github.com/lightning-resume/template-navy-react.git"
+  "navy": "https://github.com/lightning-resume/template-navy-react/archive/main.zip"
```

Comment:
why need this??

Reply:
In order to download the zip directly and avoid cloning the repo and relying on the user's git setup